analyzers/maven/maven_test.go

```diff
@@ -0,0 +1,39 @@
+package maven_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/fossas/fossa-cli/analyzers/maven"
+	"github.com/fossas/fossa-cli/module"
+	"github.com/fossas/fossa-cli/pkg"
+)
+
+func TestCustomMavenPOMFile(t *testing.T) {
+	buildTarget := "github.com/fossas/fossa-cli/cmd/fossa"
```

Comment:
I don't think this build target correctly corresponds to a Maven subproject. In this case, I think there are no subprojects (so the correct build target would be an empty string).

I'm not sure we handle the empty subproject case correctly in `DependencyTree` -- you might need to add in additional logic to avoid passing the project flag if the parameter is not provided.

Reply:
Ahhh, that is why I got confused with it.

*On it!*

Reply:
This has been resolved!